### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by deephunt3r**
## Related Issue 
- the Rules of GSSoC 21
Closes: #68

### Describe the changes you've made
I have added the rules which are to be followed by Gssoc participant
## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I have discussed with project admin before creating pr

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original                                                        | Updated
 :--------------------:                                 |:--------------------:
 original screenshot                                | updated screenshot |
 Screenshot 2021-03-17 092932 | Screenshot 2021-03-17 093044 |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,38 @@ If you're interested, why not start with:
 
 ![GSSoC Logo](https://i.imgur.com/BR9Q5Pd.png)
 
-### Rules of GSSoC 21
+## Rules of GSSoC 21
```
Comment (praveenscience) [inline README.md:49]: Please let's not change the level of heading?

File: README.md [hunk 2]
```diff
@@ -46,7 +46,38 @@ If you're interested, why not start with:
...
+    - Level1: 2 Days
+    - Level2: 3 Days
+    - Level3: 5 Days
+    
+   If you fail to make a PR within the deadline, then the issue will be assigned to another person in the queue.
+
+
+10. While making PRs, don't forget to add a description and screenshots of your work.
+
+11. Make sure your solution to any issue is better in terms of performance and other parameters in comparison to the previous work
```
Comment (praveenscience) [inline README.md:80]: No need of line spaces.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (deephunt3r) [reply README.md]: can you please check it now

**Review by praveenscience — APPROVED** [review]
👍

**Pull request merged**